exp-player/addon/components/exp-exit-survey.js

```diff
@@ -0,0 +1,194 @@
+        "type":"object",
+        "properties": {
+            "birthdate": {
+                "title":"Please confirm your child's birthdate: *"
+            },
+            "feedback": {
+                "type":"string",
+                "title":"Your feedback:",
+            },
+            "privacy": {
+                "type": "string",
+                "title": "Select the privacy level for the video of your participation: *",
+                "enum": ['Publicity and educational use (video may be publicity or educational purposes online or in the press in addition to for research purposes)', 'Scientific use only', 'In-lab use only', 'WITHDRAW your data from this session']
+            }
+        }
+    },
+    "options": {
+        "fields": {
+            "birthdate": {
+                "type": "date",
+                "manualEntry": false,
+                "validator": "required-field",
+                "message": "Please provide a complete and valid birthday.",
+                "helper": "We ask again just to check for typos during registration or people accidentally selecting a different child at the start of the study.",
+            },
+            "feedback": {
+                "type": "textarea",
+                "rows": 5,
+                "cols": 40,
+                "helper": "Do you have any ideas about how to make this study easier or more fun for families? Did you experience any technical challenges?"
+            },
+            "privacy": {
+                "type": "radio",
+                "removeDefaultNone": true,
+                "validator": "required-field",
+                "message": "Please select a privacy level for your video."
+            }
+        }
+    }   
+};
+const emailOptOutSchema = {
+    "schema": {
+        "type":"object",
+        "properties": {
+            "emailOptOut": {
+                "title":"You are currently signed up to recieve email reminders about this study."
+            }
+        }
+    },
+    "options": {
+        "fields": {
+            "emailOptOut": {
+                "type": "checkbox",
+                "rightLabel": "Opt out"
+            }
+        },
+        "form": {
+            "buttons": {
+                "finish": {
+                    "title": "Back to Lookit home page"
+                }
+            }
+        }
+    }  
+};
+
+export default ExpFrameBaseComponent.extend({
+    layout: layout,
+    type: 'exp-exit-survey',
+    meta: {
+        name: 'ExpExitSurvey',
+        description: 'Exit survey for Lookit.',
+        parameters: {
+            type: 'object',
+            properties: {
+                id: {
+                    type: 'string',
+                    description: 'A unique identifier for this item'
+                },
+                title: {
+                    type: 'string',
+                    description: 'A title for this item',
+                    default: 'Exit Survey'
+                },
+                exitThankYou: {
```

Comment:
All of them except for exitThankYou and exitMessage will come from the context that is not yet available (this PR can wait until it is though). The exitThankYou/exitMessage could be in the template, an attribute of the component, or passed in. Is there a preference?